src/Rest/GlobalCommand.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Exan\Fenrir\Rest;
+
+use Discord\Http\Endpoint;
+use Discord\Http\Http;
+use Exan\Fenrir\Parts\ApplicationCommand;
+use Exan\Fenrir\Rest\Helpers\Command\CommandBuilder;
+use Exan\Fenrir\Rest\Helpers\HttpHelper;
+use JsonMapper;
+use React\Promise\ExtendedPromiseInterface;
+
+/**
+ * @see https://discord.com/developers/docs/interactions/application-commands
+ */
+class GlobalCommand
+{
+    use HttpHelper;
+
+    public function __construct(private Http $http, private JsonMapper $jsonMapper)
+    {
+    }
```

Comment:
```suggestion
    public function __construct(private Http $http, private JsonMapper $jsonMapper) {}
```
Not sure if this is PSR but I usually do this for empy constructors

Reply:
Not PSR, plus I dont like it :stuck_out_tongue: